tests/run.mjs

```diff
@@ -83,20 +83,25 @@ async function testEnd2End() {
     try {
         await driver.get(`http://localhost:${PORT}/index.html?worstCaseCount=2&iterationCount=3`);
         await driver.executeAsyncScript((callback) => {
-            globalThis.addEventListener("JetStreamReady", callback);
+            globalThis.addEventListener("JetStreamReady", () => callback());
```

Comment:
I'm assuming the arrow function somehow fixes the serialization issue? If so it's probably worth a comment so someone doesn't come through later and undo it thinking it's an innocuous refactor.